src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -483,9 +449,12 @@ private async Task<string> GetOnlinePackagePath(SdkFeatureBand sdkFeatureBand, M
         private string GetOfflinePackagePath(SdkFeatureBand sdkFeatureBand, ManifestId manifestId, DirectoryPath? offlineCache = null)
         {
             string packagePath = Directory.GetFiles(offlineCache.Value.Value)
-                .Where(path => path.EndsWith(".nupkg"))
```

Comment:
We shouldn't chain `Where` statements. I collapsed the condition into the other `Where`.